test/v1/controllers/DeleteLossClaimControllerSpec.scala

```diff
@@ -76,6 +77,12 @@ class DeleteLossClaimControllerSpec
         val result: Future[Result] = controller.delete(nino, claimId)(fakeRequest)
         status(result) shouldBe NO_CONTENT
         header("X-CorrelationId", result) shouldBe Some(correlationId)
+
+        val detail = DeleteLossClaimAuditDetail(
+          "Individual", None, nino,  claimId, correlationId,
+          AuditResponse(NO_CONTENT, None, Some(JsObject.empty)))
```

Comment:
Should probably be 
```suggestion
          AuditResponse(NO_CONTENT, None, None))
```

otherwise will have an empty `body` object in the resulting JSON. Means you have to use the same primary constructor in the controller but I don't think that's a problem.

Reply:
The tests won't pass with the body set too None. As it is currently written, the code returns an empty JsObject when the body has no content.

Reply:
Yes - the code should change so the test passes.

If we are to output something along the lines of what is in the Txm document (although I think we have some leeway here) then either we do this or we change the AuditResponse apply method to take an Option[Either[...]], or hack the Writes to not output a body for an empty JsObject (but this is horrible).